doc-content/enterprise-only/optimizer/code-quarkus-redhat-com-proc.adoc

```diff
@@ -0,0 +1,86 @@
+[id="code-quarkus-redhat-com-proc_{context}"]
+= Creating a {QUARKUS} Maven project using code.quarkus.redhat.com
+
+[role="_abstract"]
+You can use the `code.quarkus.redhat.com` website to generate a {PLANNER} Quarkus Maven project and automatically add and configure the extensions that you want to use in your application.
+In addition, `code.quarkus.redhat.com` automatically manages the configuration parameters required to compile your project into a native executable.
+
+This section walks you through the process of generating an OptaPlanner Maven project including:
+
+ * Specifying basic details about your application.
+ * Choosing the extensions that you want to include in your project.
+ * Generating a downloadable archive with your project files.
+ * Using the custom commands for compiling and starting your application.
+
+.Prerequisites
+* You have a web browser.
+
+.Procedure
+
+. Enter the following URL in your web browser:
++
+[source]
+----
+code.quarkus.redhat.com
```

Comment:
I would replace this (starting on line 20) with the following:

1. Open https://code.quarkus.redhat.com in your web browser.

So that as a reader I can simply click or Ctrl+click the link and open it in a new tab.

Reply:
Fixed